### PR TITLE
Rollback removal of assign pm stage

### PR DESCRIPTION
**Pull request by marcuspp**
### Description of change
This removes the work done to remove the `Assign PM` from the investment flow. The work was released behind a feature flag which was never activated. 
The future plan is to add triaging in to the investment flow so these changes will no longer be required.

### To test
Test everything works the same
- Create an investment project
- On saving the prospect you should then be told what data is needed to move to `Assign PM`
- Fill in the required data
- Move project to `Assign PM`
- Fill in `Project manager` and `Project Assurance Adviser` and move to `Active`

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/investment/project/validate.py [hunk 1]
```diff
@@ -38,24 +36,15 @@ def _get_to_many_id(instance):
...
-        if self.is_streamlined_flow:
-            return Stage.active.value
-        return Stage.assign_pm.value
-
-    def get_required_fields_after_stage(self):
+    @classmethod
+    def get_required_fields_after_stage(cls):
         """Mapping from field name to the stage the field becomes required."""
-        next_stage_for_prospect = self._get_next_stage_for_prospect()
         return {
```
Comment (reupen) [inline datahub/investment/project/validate.py]: Do we still need to wrap this (and the next) dict in a class and method?

File: datahub/investment/project/test/test_validate.py [hunk 2]
```diff
@@ -391,11 +383,6 @@ class TestValidationConfig:
 
     EXPECTED_STAGE = constants.InvestmentProjectStage.assign_pm.value
 
-    def test_get_next_stage_for_prospect(self):
-        """Tests get next investment project stage after prospect."""
-        validation_config = InvestmentProjectStageValidationConfig()
-        assert validation_config._get_next_stage_for_prospect() == self.EXPECTED_STAGE
-
     def test_required_fields_after_stage(self):
```
Comment (marcofucci) [inline datahub/investment/project/test/test_validate.py]: The docstring in `test_get_desired_stage_order` needs updating.

File: datahub/investment/project/validate.py (hunk 1 above)
Comment (marcuspp) [reply datahub/investment/project/validate.py]: Technically not but I think it keeps things a bit neater leaving it as is.

Comment (reupen) [reply datahub/investment/project/validate.py]: The tests aren't really doing anything now (the assertions are just testing fixed values, so I think they should at least go: https://github.com/uktrade/data-hub-leeloo/blob/6860ce86ba09e636e93fa55eea08b15ccfd0f061/datahub/investment/project/test/test_validate.py#L381

**Pull request merged**